README.rst

```diff
@@ -36,11 +36,39 @@ https://osprey.readthedocs.io.
 Submit bug reports, questions and feature requests at
 https://github.com/nikola-rados/osprey/issues
 
+Installation
+------------
+
+Clone the repo onto the target machine. Python installation should be done in a python3 virtual environment created
+and activated as follows (second `venv` can be replaced with environment name of your choice):
+
+.. code:: bash
+
+   $ python3 -m venv venv
+   $ source venv/bin/activate
+   (venv) $ pip install -i https://pypi.pacificclimate.org/simple/ -r requirements.txt -r requirements_dev.txt
+   (venv) $ pip install -e .
```

Comment:
The installation instructions are already explained in the `docs`. I don't mind if we move the general installation method out here thought. Especially since we have to make some changes to the code.

We want the `pip` instructions to match the `docs`:
```
(venv)$ pip install -r requirements.txt
(venv)$ pip install -e .[dev]